lib/auth/init.go

```diff
@@ -822,16 +821,26 @@ func checkResourceConsistency(clusterName string, resources ...types.Resource) e
 			// check that signing CAs have expected cluster name and that
 			// all CAs for this cluster do having signing keys.
 			seemsLocal := r.GetClusterName() == clusterName
+
 			var hasKeys bool
-			_, err := sshSigner(r)
+			var signerErr error
+			switch r.GetType() {
+			case types.HostCA, types.UserCA:
+				_, signerErr = sshSigner(r)
+			case types.JWTSigner:
+				_, signerErr = services.GetJWTSigner(r, clockwork.NewRealClock())
```

Comment:
Note to reviewers: this block changed in relation to master, since keystore integration hasn't landed here yet.